tutorial/tutorial-gabc-03.html

```diff
@@ -174,15 +199,19 @@
 <img src="kyrie-17-15.png"/>
 </div>
 
+<p>If you don't see the difference, look particularly at the initial "K" and note where the lower diaganol branches off from the upper one.</p>
+
 <p>The best-looking free font for gregorio is a matter of taste, but <code>kpfonts</code> is also a good choice (<code>\usepackage{kpfonts}</code>).</p>
 
+<p>For a list of free fonts which are readily available for <span class="tex">T<span class="epsilson">e</span>X</span> see <a href="http://www.tug.dk/FontCatalogue/">The <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> Font Catalogue.</a>  Other fonts, including most of the ones already installed on your system, can also be used, but we'll leave it up to you to explore the use of other fonts on your own.</p>
```

Comment:
I think the best would be to make users use fontspec, but I don't think there currently is a catalogue for it (nor do I think there will be), as it doesn't make much more sense than a list of fonts for OpenOffice: these softwares just use whatever font is on the system. But maybe we could describe both: the font catalogue and a sentence about system fonts?